pkg/testutil/testutil.go

```diff
@@ -494,8 +501,8 @@ func CreateTestKopiaRepository(cli kubernetes.Interface, rs *v1alpha1.Repository
 	)
 }
 
-func GetTestKopiaRepositoryServerCR(namespace string) *crv1alpha1.RepositoryServer {
-	repositoryServer := &crv1alpha1.RepositoryServer{
+func GetTestKopiaRepositoryServerCR(namespace string) crv1alpha1.RepositoryServer {
```

Comment:
Why do we need this change? Can we separate it out in a different PR if it's not related to this PR?

Reply:
since we are using pointer for default repository server spec, if we make changes to its spec in 1 test, other tests are also impacted. The test for this PR was also failing because of that. Hence I had added this change. I will add it to another PR which will be used as a base in this PR/

Reply:
Added it in separate PR - https://github.com/kanisterio/kanister/pull/2170